dev_guide/quota.adoc

```diff
@@ -146,16 +238,20 @@ metadata:
 spec:
   hard:
     configmaps: "10" <1>
-    persistentvolumeclaims: "4" <2>
-    replicationcontrollers: "20" <3>
-    secrets: "10" <4>
-    services: "10" <5>
+    openshift.io/imagestreamtags: "10" <2>
```

Comment:
please move all things related to imagestreamtags and imagestreamimages into a separate example quota file specific to image quota.